crates/bevy_state/src/app.rs

```diff
@@ -60,15 +60,15 @@ pub trait AppExtStates {
 }
 
 /// Separate function to only warn once for all state installation methods.
```

Comment:
```suggestion
/// Helper function to panic if [`StatesPlugin`] is not installed.
```
Or other wording that makes this comment not outdated.